unit-10/homework/index.js

```diff
@@ -0,0 +1,49 @@
+// Exercise-1 In một bảng nhân của một số bất kỳ với các số từ 1 tới 10 và hiển thị kết quả.
+const calcMultipliers = num => {
+  for (let i = 1; i < 11; i++) {
+    console.log (num * i)
+    }
```

Comment:
Xem dấu cách
Và format in ra không đúng yêu cầu bài toán
```suggestion
  for (let i = 1; i < 11; i++) {
    console.log(num * i)
  }
```

Nếu đúng yêu cầu thì phải như thế này:
```javascript
console.log(`${num} x ${i} = ${num * i}`);
```